workflows/toast_sim_ground.py

```diff
@@ -226,6 +226,85 @@ def use_full_pointing(job):
     return full_pointing
 
 
+def select_pointing(job, args):
```

Comment:
This seems like a lot of code to duplicate in all workflows that support multiple pixelizations.  Fine for now but we should think of how to avoid the duplication.

Reply:
If there is truly code duplication, then I agree.  I can imagine that individual projects would put these high-level, common workflow functions into their own packages (for example, `sotodlib`).  If we have more than one workflow within toast itself using identical code then we could move it into a subpackage.  However, individual projects will likely need their own sorts of helper functions like this.  For example, an instrument on the LMT is not going to use healpix pointing for small patch observations and would not have that in their workflows at all.  Similarly individual experiments working with real data will likely have their own operators for detector pointing models and stokes weights that deal with more complicated pointing reconstruction, HWP customization, etc.  So building a generic version of high-level workflow helper functions is probably specific to an experiment (at least for real data).